README.md

```diff
@@ -471,6 +471,47 @@ With the flag `--keep_going=N` Jazzer continues fuzzing until `N` unique stack t
 Particular stack traces can also be ignored based on their `DEDUP_TOKEN` by passing a comma-separated list of tokens
 via `--ignore=<token_1>,<token2>`.
 
+### Export coverage information
+
+The internally gathered JaCoCo coverage information can be exported in human-readable and JaCoCo dump format.
+These can help identify code areas that can not be reached and perhaps need improvements to make them more accessible
+for the fuzzer.
```

Comment:
I think it's important to offer clear guidance for this case. How about something like:

```suggestion
These can help identify code areas that have not been covered by the fuzzer and thus may require more comprehensive fuzz targets or a more extensive initial corpus to reach.
```

Reply:
done